### PR TITLE
Issue9

### PR DESCRIPTION
**Pull request by singh-yashwant**
added the --expire argument
this argument when called while running the script will invoke the newly added function "expire_overdue_todos" which will delete all the older todos from the database irrespective of their status.

### CODE REVIEW

**Comment by thescriptninja** [comment]
Looks good. Just update the docstring of the `expire_overdue_todos` function so that it is in the same format of the other functions. Otherwise, it is fine. You can push the changes to the `issue9` branch on your fork. Thanks!
Small suggestion: Please be more descriptive in the PR title, as to what it will do when it is merged. Keep that in mind in your next PR. :smiley:

File: pytodo/modules.py [hunk 1]
```diff
@@ -130,3 +130,13 @@ def expire_todos():
     for todo in collection.find():
         if datetime.now() - todo["created"] > timedelta(hours=1) and todo["is_done"]:
             delete_by_id(todo["_id"])
+
+
+def expire_overdue_todos(todos):
+    """
+    This method when called will automatically remove all overdue todos \n
```
Comment (thescriptninja) [inline pytodo/modules.py]: Please change this to `Removes all overdue todos irrespective of their status`.

**Comment by singh-yashwant** [comment]
Sure will update it
Thanks

**Comment by thescriptninja** [comment]
I have made the changes myself since it was minor and it was holding up another PR. Thanks for the contribution @singh-yashwant ! Awesome work!

**Pull request merged**